Jenkinsfile

```diff
@@ -212,7 +196,7 @@ pipeline {
                     }
                 }
                 stage('Clang Format') {
-                    agent{ label rocmnode("rocmtest") }
+                    agent{ label rocmnode("nogpu") }
```

Comment:
What happens if only have nodes with gpus? Ideally, these stages should be moved to github hosted runners.

Reply:
Any node that has `nogpu` is allowed to run such tests. Currently, these are all Vega nodes and none of MI100s.

Reply:
I can rename it to `allow_nogpu` but this is shorter...

Reply:
This is noted in the the PR description.